src/Routing/ResolvedRoute.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Tnapf\Router\Routing;
+
+use stdClass;
+
+class ResolvedRoute {
+    public function __construct(
+        public readonly Route $route, 
+        public readonly stdClass $args
+    ) {}
+}
```

Comment:
File should have a unix line ending it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces